server/src/resolvers/mapping/utils.ts

```diff
@@ -61,8 +64,15 @@ export const buildInputsQuery = (
     return input
   })
 
-export const buildAttributesQuery = (
-  attributes: AttributeWithInputs[],
+export const buildCommentQueryPreV7 = (
+  comment: string,
+): CommentCreateWithoutAttributeInput => ({
+  content: comment,
+  author: { connect: { email: 'admin@arkhn.com' } },
+})
+
+export const buildAttributesQueryPreV7 = (
```

Comment:
Do you think it would be more readable if we had only one function and pass it also the version?

Reply:
yes it would, i'll do that. But anyway I think we'll get rid of older versions soon to avoid having to maintain a bunch of versions at the same time

Reply:
oh no actually I can't because the function signatures are not the same. Before v7, `attribute.comments` is a `string`, after it's a `IComment`